src/main/java/dataProviders/ConfigFileReader.java

```diff
@@ -58,7 +58,7 @@ public long getImplicitlyWait() {
 	public String getApplicationUrl() {
 		String url = properties.getProperty("applicationURL");
 		if(url != null) {
-			return "https://" + url;
+			return "https://www." + url;
```

Comment:
Does the Lucency URL contains string www in the URL?
like https://www.stg.lucency.com